unittests/tools/test_edgescan_parser.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from django.test import TestCase
+
+from dojo.tools.edgescan.parser import EdgescanParser, ES_SEVERITIES
+from dojo.models import Test, Finding
+
+
+class TestEdgescanParser(TestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        with open("unittests/scans/edgescan/no_vuln.json") as testfile:
+            parser = EdgescanParser()
+            findings = parser.get_findings(testfile, Test())
+            self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_findings(self):
+        with open("unittests/scans/edgescan/one_vuln.json") as testfile:
+            parser = EdgescanParser()
+            findings = parser.get_findings(testfile, Test())
+            self.assertEqual(1, len(findings))
+            finding = findings[0]
+            testfile.seek(0)
+            payload = json.loads(testfile.read())[0]
+            self.assertEqual(finding.title, payload["name"])
+            self.assertEqual(finding.date, payload["date_opened"][:10])
+            self.assertEqual(finding.cwe, int(payload["cwes"][0][4:]))
+            self.assertEqual(finding.cve, str(payload["cves"]))
```

Comment:
I took a look at the data of the unit tests and the CVE are weird.
Like this:
![image](https://user-images.githubusercontent.com/1694940/148683340-80544a78-85fe-4b91-9803-f817668ad4d0.png)
CVE are more like `CVE-2021-44228` (see https://www.cvedetails.com/cve/CVE-2021-44228/)

Reply:
Fixed in 4977b14.